src/third-party/README.md

```diff
@@ -0,0 +1,2 @@
+These are packages that we depend upon. They may need to be trimmed, all .cpp
```

Comment:
No, we need a subset of the repo, if we include the whole thing arduino will try to compile their tests too. I cannot find a way to submodule or subtree only part of a repo.

Also since they do actual releases it may be nicer to stay synced to actual releases.